src/debugger.ts

```diff
@@ -192,16 +209,19 @@ export class Debugger
     const configuration = session.configuration;
     const workspaceFolder = configuration.targetFolder;
     const cwd = workspaceFolder.path;
+    const port =
```

Comment:
Using UNIX sockets doesn't work on Windows, but we can easily spawn the debugger with a port.